.gitignore

```diff
@@ -17,4 +17,5 @@ build/regulation-worker
 *.out.*
 *.out
 coverage.txt
-coverage.html
\ No newline at end of file
+coverage.html
+*.orig
```

Comment:
what is `*.orig` ?

Reply:
leftovers from [git-mergetool](https://git-scm.com/docs/git-mergetool#Documentation/git-mergetool.txt-mergetoolkeepBackup)